pom.xml

```diff
@@ -229,8 +230,8 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-compiler-plugin</artifactId>
         <configuration>
-          <source>1.6</source>
-          <target>1.6</target>
+          <source>${jdk.version}</source>
+          <target>${jdk.version}</target>
```

Comment:
Can you change property  to *java.version* as mentioning jdk does not seem to be good for target?